### ISSUE TITLE
[Mint] Mint contact information not encoded correctly according to NUT-09

### ISSUE THREAD

**Issue by callebtc** [cashubtc/cashu#180]
[NUT-09](https://github.com/cashubtc/nuts/blob/main/09.md) wants this

```json
{
...,
"contact": [
    ["email", "contact@me.com"],
    ["twitter", "@me"],
    ["nostr" ,"npub..."]
  ]
}
```

whereas the `.env.example` entry is 

```bash
MINT_INFO_CONTACT=["Email: contact@me.com", "Twitter: @me", "Nostr: npub..."]
```

Response of `https://8333.space:3338/info` is empty:

```bash
... "contact":[]
```

**Comment by xphade** [comment]
I just thought I could take this as a first task but I think you fixed this already in #189, right? 🙂

**Comment by callebtc** [comment]
Youre right! Thanks!
